packages/did-document/src/did-document.ts

```diff
@@ -1,36 +1,14 @@
-import { Keyring } from '@verida/keyring'
 import { ServiceEndpoint, Service } from 'did-resolver'
 import EncryptionUtils from '@verida/encryption-utils'
 import { VerificationMethod } from 'did-resolver'
-import { knownNetworks, strip0x } from './helpers'
-import { BigNumber } from '@ethersproject/bignumber'
-import { computeAddress } from "@ethersproject/transactions";
-import { getAddress } from "@ethersproject/address";
+import { strip0x } from './helpers'
 import { IDIDDocument, IKeyring, SecureContextEndpoints, SecureContextEndpointType, VeridaDocInterface, VerificationMethodTypes } from '@verida/types'
+import { interpretIdentifier } from '@verida/vda-common'
+import { mapDidNetworkToBlockchainAnchor } from '@verida/vda-common'
+import { BLOCKCHAIN_CHAINIDS } from '@verida/vda-common'
+import { BlockchainAnchor } from '@verida/types'
 const _ = require('lodash')
```

Comment:
Out of scope for this PR but such `require` prevents tree-shaking and loads the whole library into the build down the line, even if only one or two methods are actually used.
`loadsh` provides named exports/imports